.github/workflows/build-and-deploy.yml

```diff
@@ -22,12 +22,12 @@ on:
 
 jobs:
   build:
-    uses: ZTL-ARTCC/Current_Website/.github/workflows/build.yml@main
+    uses: ZTL-ARTCC/Current_Website/.github/workflows/build.yml@laravel-10
```

Comment:
We'll just need to make sure this gets changed back before merging into main

Reply:
Yes, absolutely. I'll leave this comment unresolved as a reminder.

Reply:
Updated GHA target to main